sky/data/storage_utils.py

```diff
@@ -162,3 +203,20 @@ def get_excluded_files_from_gitignore(src_dir_path: str) -> List[str]:
                     to_be_excluded += '*'
                 excluded_list.append(to_be_excluded)
     return excluded_list
+
+
+def get_excluded_files(src_dir_path: str) -> List[str]:
+    # TODO: this could return a huge list of files,
+    # should think of ways to optimize.
+    """ List files and directories to be excluded.
+    """
```

Comment:
style: keep docstr in one line
```suggestion
    """Lists files and directories to be excluded."""
```